include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
Uhm, yeah, do feel different about it. Especially in CppUtest tests, including StandardCLibrary avoids a direct dependency with StandardC library, which is great :)